library/src/main/java/com/android/volley/RequestQueue.java

```diff
@@ -203,6 +206,14 @@ public VolleyTimingLogger getTimingLogger() {
         return mTimingLogger;
     }
 
+    public void setRequestLogger(RequestLogger logger) {
```

Comment:
This (and the `TimingLogger`) should be added to the constructor, and not assigned via setter, like the other dependencies.